Mage.Sets/src/mage/cards/m/Malanthrope.java

```diff
@@ -52,6 +53,12 @@ public Malanthrope copy() {
 
 class MalanthropeEffect extends OneShotEffect {
 
+    private static final FilterCard filter = new FilterCard();
```

Comment:
Minor nitpick, but you should use the already made static filter:
https://github.com/magefree/mage/blob/c8c663b9764f7bc2411d8dddf4d6eadc86770cb7/Mage/src/main/java/mage/filter/StaticFilters.java#L77

Reply:
Done.